.github/ISSUE_TEMPLATE/BUG-ISSUE.yml

```
@@ -2,68 +2,86 @@ name: Bug Report
 description: Create a report to help us improve
 labels: ["Type: Bug"]
 body:
+  - type: markdown
+    attributes:
+      value: |
+        Note: Internal developers should consider opening a private issue if there are sensitive details that would be helpful to include
+        Links to private resources (ie logs, slack conversations, etc.) can be shared in public issues
   - type: dropdown
+    id: product
+    attributes:
+      label: Gloo Edge Product
+      description: Which Edge product are you using?
+      options:
+        - Open Source
+        - Enterprise
+    validations:
+      required: true
+  - type: input
     id: version
     attributes:
       label: Gloo Edge Version
-      description: What version of Gloo Edge are you using?
-      options:
-        - 1.15.x (beta)
-        - 1.14.x (latest stable)
-        - 1.13.x
-        - 1.12.x
-        - 1.11.x
+      description: What version of your product are you using? Please specify a specific patch version
+      placeholder: eg v1.14.2
     validations:
       required: true
-  - type: dropdown
+  - type: input
     id: kube-version
     attributes:
       label: Kubernetes Version
       description: What version of Kubernetes are you using with Gloo Edge?
-      options:
-        - 1.27.x
-        - 1.26.x
-        - 1.25.x
-        - 1.24.x
-        - 1.23.x
-        - 1.22.x 
-        - 1.21.x
-        - 1.20.x
-        - 1.19.x
-        - 1.18.x
-        - 1.17.x 
-        - 1.16.x
-        - 1.15.x
-        - 1.14.x
-        - 1.13.x
-        - 1.12.x
+      placeholder: eg v1.26.6
   - type: textarea
     id: describe-bug
     attributes:
       label: Describe the bug
-      placeholder: Add a clear and concise description of what the bug is
+      description: A clear and concise description of what the bug is
+      placeholder: |
+        The x resource enters a failed state when I set field foo to bar
+    validations:
+      required: true
+  - type: textarea
+    id: expected-behavior
+    attributes:
+      label: Expected Behavior
+      description: A clear and concise description of what is expected to happen
+      placeholder: When I performed x, I expected y to happen
```

Maybe the placeholder can contain a config snippet and failure status message to give a concrete example of what we mean by expected behavior.

---

Could you elaborate? 

Do you mean something like:

```suggestion
      placeholder: |
          Resource x should be accepted.
          Specifically the CR status should look like:
          ```
          status:
            statuses:
              gloo-system:
                reportedBy: gloo
                state: 1
          ```
```

---

Yeah like that or perhaps a piece of good looking configuration would make a little more sense like
```
spec:
  virtualHost:
    domains:
    - '*'
    routes:
    - matchers:
      - exact: /all-pets
     ```

---

I think it might make more sense to have something like this in the `describe-bug` section
Users have have config to share, but I doubt we'd want them to be prescriptive about what it _should_ look like per se